### PR TITLE
zh trans ：website/content/en/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md

### PR DESCRIPTION
**Pull request by zhangx501**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For pull requests on Chinese localization, set Base Branch to release-1.14
>
> For pull requests on Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by zhangx501** [comment]
/assign @sataqiu
The file I've translated in the first time has problem. The link can't get the file. "https://deploy-preview-15765--k8s-v1-14.netlify.com/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes" Now I upload the file again. It can't get approved. Could you help me deal with this?

**Comment by SataQiu** [comment]
Thanks for your contribution. @zhangx501
You can get it by this link: https://deploy-preview-15813--kubernetes-io-master-staging.netlify.com/blog/2016/02/kubernetes-community-meeting-notes/ 
However, in fact, we don't recommend translating such old articles. Even all blog posts are translated with low priority. We hope you to translate more new content, preferably the core content, rather than blog posts. Guide: https://github.com/k8smeetup/k8s-official-translation

File: content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+title: " Kubernetes 社区会议记录 - 20160204 "
```
Comment (SataQiu) [inline content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md]:
```suggestion
title: "Kubernetes 社区会议记录 - 20160204"
```

File: content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+title: " Kubernetes community meeting notes - 20160204 "
+date: 2016-02-09
+slug: kubernetes-community-meeting-notes
+url: /blog/2016/02/Kubernetes-Community-Meeting-Notes
+---
+-->
+<!--
+####  February 4th - rkt demo (congratulations on the 1.0, CoreOS!), eBay puts k8s on Openstack and considers Openstack on k8s, SIGs, and flaky test surge makes progress.
+-->
+####  2 月 4 日 - rkt演示（祝贺 1.0 版本， CoreOS！）， eBay 将 k8s 放在 Openstack 上并认为 Openstack 在 k8s， SIG 和片状测试激增方面取得了进展。
```
Comment (SataQiu) [inline content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md]:
```suggestion
####  2 月 4 日 - rkt 演示（祝贺 1.0 版本， CoreOS！）， eBay 将 k8s 放在 Openstack 上并考虑把 Openstack 放在 k8s 上， SIG 和片状测试激增（flaky test surge）方面取得了进展。
```

File: content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+<!--
+The Kubernetes contributing community meets most Thursdays at 10:00PT to discuss the project's status via a videoconference. Here are the notes from the latest meeting.
+-->
+Kubernetes 贡献社区在每周四 10:00 PT 开会,通过视频会议讨论项目状态。以下是最近一次会议的笔记。
```
Comment (SataQiu) [inline content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md]:
```suggestion
Kubernetes 贡献社区在每周四 10:00 PT 开会，通过视频会议讨论项目状态。以下是最近一次会议的笔记。
```

File: content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        * [note: audio is garbled at points]
+        * Discussion about integration w/ k8s & rkt
+        * rkt community sync next week: https://groups.google.com/forum/#!topic/rkt-dev/FlwZVIEJGbY
+        * Dawn Chen:
+            * The remaining issues of integrating rkt with kubernetes: 1) cadivsor 2) DNS 3) bugs related to logging
+            * But need more work on e2e test suites
+-->
+* 记录员：Rob Hirschfeld
+* 演示视频（20 分钟）：CoreOS rkt + Kubernetes[Shaya Potter]
+    * 期待在未来几个月内看到与rkt和k8s的整合（“rkt-netes”）。 还没有集成到 v1.2版本中。
```
Comment (SataQiu) [inline content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md]:
```suggestion
    * 期待在未来几个月内看到与 rkt 和 k8s 的整合（“rkt-netes”）。 还没有集成到 v1.2 版本中。   
```

File: content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+        * Discussion about integration w/ k8s & rkt
+        * rkt community sync next week: https://groups.google.com/forum/#!topic/rkt-dev/FlwZVIEJGbY
+        * Dawn Chen:
+            * The remaining issues of integrating rkt with kubernetes: 1) cadivsor 2) DNS 3) bugs related to logging
+            * But need more work on e2e test suites
+-->
+* 记录员：Rob Hirschfeld
+* 演示视频（20 分钟）：CoreOS rkt + Kubernetes[Shaya Potter]
+    * 期待在未来几个月内看到与rkt和k8s的整合（“rkt-netes”）。 还没有集成到 v1.2版本中。   
+    * Shaya 做了一个演示（8分钟的会议视频参考）
```
Comment (SataQiu) [inline content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md]:
```suggestion
    * Shaya 做了一个演示（8 分钟的会议视频参考）
```

File: content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+        * rkt的 CLI 显示了旋转容器
```
Comment (SataQiu) [inline content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md]:
```suggestion
        * rkt 的 CLI 显示了旋转容器
```

File: content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+            * [注意：音频在点数上是乱码]
+        * 关于 k8s&rkt 整合的讨论
```
Comment (SataQiu) [inline content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md]:
```suggestion
        * 关于 k8s & rkt 整合的讨论
```

File: content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+        * 下周 rkt 社区同步：https://groups.google.com/forum/#!topic/rkt-dev/FlwZVIEJGbY
+        * Dawn Chen:
+            * 将 rkt 与 kubernetes 集成的其余问题：1）cadivsor 2） DNS 3）与日志记录相关的错误
```
Comment (SataQiu) [inline content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md]:
```suggestion
            * 将 rkt 与 kubernetes 集成的其余问题：1）cadivsor 2）DNS 3）与日志记录相关的错误
```

File: content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+    *  想要创建必要的基础设施，使测试设置不那么薄弱
+    *  想要将测试开始（单次或完整）与 Jenkins分离
+    *  目标是指出你有一个可以指向任何集群的脚本
+    *  演示包括 Google 内部视图 - 努力尝试获取外部视图。
+    *  希望能够收集测试运行结果
+    *  Bob Wise 不赞同在 v1.3 版本进行测试方面的基础设施建设。
+    *  关于测试实践的长期讨论…
+       * 我们希望在多个平台上进行测试的共识。
+       * 为测试报告提供一个全面转储会很有帮助
+       * 可以使用"phone-home"收集异常
```
Comment (SataQiu) [inline content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md]:
```suggestion
       * 可以使用 "phone-home" 收集异常
```

File: content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+
+
+<!--
+* 1.2 Release Watch
+* CoC [Sarah]
+* GSoC [Sarah]
+-->
+* 1.2发布观察
```
Comment (SataQiu) [inline content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md]:
```suggestion
* 1.2 发布观察
```

File: content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+* CoC [Sarah]
+* GSoC [Sarah]
+
+<!--
+To get involved in the Kubernetes community consider joining our [Slack channel][2], taking a look at the [Kubernetes project][3] on GitHub, or join the [Kubernetes-dev Google group][4]. If you're really excited, you can do all of the above and join us for the next community conversation -- February 11th, 2016. Please add yourself or a topic you want to know about to the [agenda][5] and get a calendar invitation by joining [this group][6].   
+-->
+要参与 Kubernetes 社区，请考虑加入我们的[Slack 频道][2]，查看 GitHub上的 [Kubernetes 项目][3]，或加入[Kubernetes-dev Google 小组][4]。如果你真的很兴奋，你可以完成上述所有工作并加入我们的下一次社区对话-2016年2月11日。请将您自己或您想要了解的主题添加到[议程][5]并通过加入[此组][6]来获取日历邀请。
```
Comment (SataQiu) [inline content/zh/blog/_posts/2016-02-00-Kubernetes-Community-Meeting-Notes.md]:
```suggestion
要参与 Kubernetes 社区，请考虑加入我们的 [Slack 频道][2]，查看 GitHub上的 [Kubernetes 项目][3]，或加入 [Kubernetes-dev Google 小组][4]。如果你真的很兴奋，你可以完成上述所有工作并加入我们的下一次社区对话 - 2016 年 2 月 11 日。请将您自己或您想要了解的主题添加到[议程][5]并通过加入[此组][6]来获取日历邀请。
```

**Comment by zhangx501** [comment]
/assign @SataQiu
Thank you for your advice. I get the preview link. I just want to know how can I get the file I uploaded that I can correct the writting error. Then I can closed is translation mission. Also I accepted two other mission which are 2016 post notes two. How can I delete them or close them? Thank you.

**Comment by SataQiu** [comment]
图片
1. The PR is automatically built by netlify, you can preview the website by click `Details`. 
2. You can unassign yourself. If you cannot do it, please tell me the issues which you have accepted.

**Comment by markthink** [comment]
@zhangx501 需要解决一下冲突..

**Pull request closed without merging**